doc/papers/joss/paper.md

```diff
@@ -0,0 +1,115 @@
+---
+title: 'libCEED: An open-source library for efficient high-order operator evaluations'
+tags:
+  - high-performance computing
+  - high-order methods
+  - finite elements
+  - spectral elements
+  - matrix-free
+authors:
+  - name: Jed Brown
+    orcid: 0000-0002-9945-0639
+    affiliation: 1
+  - name: Jeremy Thompson
+    affiliation: 1
+  - name: Valeria Barra
+    orcid: 0000-0003-1129-2056
+    affiliation: 1
+  - name: XXXX TODO Many more authors to be added XXXX
+    affiliation: 2
+affiliations:
+ - name: University of Colorado Boulder
+   index: 1
+ - name: Lawrence Livermore National Laboratory
+   index: 2
+date: XXXX TODO write date XXXX
+bibliography: paper.bib
+
+# Optional fields if submitting to a AAS journal too, see this blog post:
+# https://blog.joss.theoj.org/2018/12/a-new-collaboration-with-aas-publishing
+aas-doi: 10.3847/xxxxx <- update this with the DOI from AAS once you know it.
+aas-journal: Astrophysical Journal <- The name of the AAS journal.
+---
+
+# Summary
+
+High-order numerical methods are widely used in Partial Differential Equation (PDE)
+solvers, but software packages that provide high-performance implementations have
+often been special-purpose and intrusive. ``libCEED`` is an open-source light-weight
+library that offers a purely algebraic interface for efficient operator evaluation
+and matrix-free preconditioning ingredients. libCEED supports run-time selection of
+implementations tuned for a variety of computational architectures, including CPUs and
+GPUs, and can be unobtrusively integrated in new and legacy software to provide portable
+performance. We introduce ``libCEED``’s conceptual framework and interface, and show examples of its integration with other packages, including PETSc and MFEM.
```

Comment:
Do we want to mention Nek as well? We aren't highly integrated, but we have examples and its a well known code.

Reply:
Sure, please feel free to make all edits and give contribution as you like. I'm not going to be the only one authoring this

Reply:
I wanted to discuss this one, since we don't have tight Nek integration like we do with MFEM and PETSc. I could do either way on including, leaning towards yes?

Reply:
I would say, yes. And we don't need to be specific in this sentence, since it says "including". Or it could be something like:  
```suggestion
performance. We introduce ``libCEED``’s conceptual framework and interface, and show examples of its integration with other packages, such as PETSc, MFEM, and Nek5000.
```

Reply:
I like that.